### PR TITLE
Feature/revpi 2832/rev pi autodiscovery

### PR DESCRIPTION
**Pull request by maosswald**

### CODE REVIEW

**Review by naruxde — CHANGES_REQUESTED** [review]
Your avahi service is no systemd-unit file, so wrong directory to place in revpi-tools.
Did you think about the file name `revpi.service`? We want to announce SSH with avahi.

Did you check the [avahi-daemon](https://packages.debian.org/buster/avahi-daemon) package? Maybe they offer examples for default services like ssh?

File: systemd/CMakeLists.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 install(FILES pileft-quirks.service piright-quirks.service hdmi-disable.service
   DESTINATION /lib/systemd/system
 )
+
```
Comment (naruxde) [inline systemd/CMakeLists.txt:4]:
```suggestion
```

Comment (naruxde) [reply systemd/CMakeLists.txt:4]: Just drop changes in this file, because you do not have to touch it for this commit.

File: CMakeLists.txt [hunk 2]
```diff
@@ -4,6 +4,7 @@ project(revpi-tools)
 
 include(GNUInstallDirs)
 
+add_subdirectory(avahi-service)
```
Comment (naruxde) [inline CMakeLists.txt:7]:
```suggestion
add_subdirectory(avahi-services)
```

I would add an 's' so you reference the meaning to /etc/avahi/services folder. Maybe we add more files here in the future.

File: avahi-service/revpi.service [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" standalone='no'?>
```
Comment (naruxde) [inline avahi-service/revpi.service:1]: Yesterday you told us, you would NOT use the name `revpi.service`. I would suggest to rename the file to `revpi-ssh.service`. The prefix could be important and indicates that it is our file.

Other files form web status could be `revpi-http.service` or things like that.

**Review by naruxde — CHANGES_REQUESTED** [review]
Your commit message should use other head text und needs a message. What will this commit do.

**Comment by RamiGspo** [comment]
Why closed?

**Comment by maosswald** [comment]
Because it was only a test to learn git look at jira ticket theres a better solution than a pull request

**Comment by RamiGspo** [comment]
@maosswald 
Maybe I am missing something, because I did not find any Information in your Jira Ticket...

**Comment by RamiGspo** [comment]
@maosswald I used your files to continue with your Ticket.
The new PR ist here: [PR#30](https://github.com/RevolutionPi/revpi-tools/pull/30)
Maybe you want to do the other part for WebStatus...

**Pull request closed without merging**